docs/concepts/unions.md

```diff
@@ -174,11 +174,6 @@ print(user_03_uuid.int)
 #> 275603287559914445491632874575877060712
 ```
 
-!!! tip
```

Comment:
Wanted to first remove the "required fields" reference, but turns out there's no `Optional` usage above, so removed completely.